sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -180,6 +180,38 @@ abstract class JdbcDialect extends Serializable with Logging {
     statement.executeUpdate(s"CREATE TABLE $tableName ($strSchema) $createTableOptions")
   }
 
+  /**
+   * Returns an Insert SQL statement for inserting a row into the target table via JDBC conn.
+   *
+   * @param table The name of the table.
+   * @param rddSchema The schema of the row that will be inserted.
+   * @param tableSchema The schema of the table.
```

Comment:
Since we are making it an API, we should make sure it's reasonable. As a JDBC dialect, why do I need to care about RDD schema? Shouldn't Spark transform the input query to match the table schema?